### ISSUE TITLE
Which OMIM genes do the BED files stem from?

### ISSUE THREAD

**Issue by Hoeze** [imgag/ngs-bits#596]
Hi, I was wondering from which genes these BED files were generated from:
https://github.com/imgag/ngs-bits/tree/master/src/MappingQC/resources

**Comment by marc-sturm** [comment]
Hi Florian,

unfortunately the original list of the 440 genes was lost in a storage failure.

We selected 10 OMIM genes per chromosome arm of the autosomes at random.
For each gene we used the gene locus (exons and introns) plus 5000 flanking bases.

I tried to re-construct the gene list based on the coordiantes and the current version of OMIM:
[region_genes.txt](https://github.com/user-attachments/files/17814745/region_genes.txt)

The reconstruction is not perfect, but I did not have time to manually curate the errors:
- some regions have two or more overlapping OMIM genes
- some genes no longer have a OMIM phenotype and thus are not annoated, e.g. PPARA 

Best,
  Marc
